recipes/cythonbiogeme/setup.cfg.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.cfg
++++ setup.cfg
+@@ -108,8 +108,6 @@
+ 	src/cythonbiogeme/cpp/bioGhFunction.cc
+ include_dirs = eval(__import__('numpy').get_include())
+ language = c++
+-extra_compile_args = -std=c++11 -DMS_WIN64
+-extra_link_args = -std=c++11 -lpthread -mms-bitfields -mwindows
+ 
+ [egg_info]
+ tag_build = 
```

Comment:
These flags will only work with mingw, not msvc.

Reply:
But, we changed to compile with {{ compiler('m2w64_cxx') }}  or not?

Reply:
I see you did that. I"ll have to look at the compile errors.